tutorials/editor/workspace_selector.rst

```diff
@@ -0,0 +1,25 @@
+.. _doc_workspace_selector:
+
+Workspace selector
```

Comment:
Let's check if this has an official name in the editor itself, or if it was named a "workspace selector" either in this PR or in a prior docs PR.

I think the name is a good one, but by adding a page here we're formally naming it.